src/components/treeActivity/index.tsx

```diff
@@ -57,6 +63,27 @@ const CenteredPagination = styled(Pagination)`
   }
 `;
 
+const DeleteActivityButton = styled(LinkButton)`
+  color: white;
+  margin: 10px;
+  padding: 0px 10px;
+  background: #d07b69;
```

Comment:
🔧 Could we be able to turn `#d07b69` into a constant (we can call it light red or pink or salmon or something) and define it in `../../utils/colors` (similar to the other color constants)? Same for line 74 as well.